ui/docs/reference/workers/generic-worker/support-tiers.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Support Tiers
+order: 35
+---
+
+# Support Tiers
+
+While Generic-Worker can run on a wide variety of platforms, the level of support for those platforms varies.
+Support levels are categorized into tiers, each bringing a different set of guarantees.
+This model is based on [Rust's support tiers](https://forge.rust-lang.org/release/platform-support.html).
+
+## Tier 1
+
+Tier one platforms are "guaranteed to worl".
```

Comment:
"guaranteed to worl" sounds like these platforms are guaranteed to spiral out of control. Maybe we should keep the typo

Reply:
'take 'em for a spin!"